.github/workflows/dbt_run.yml

```diff
@@ -16,6 +16,30 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Check for base_trades changes
```

Comment:
```suggestion
      - name: Check for dex_base_trades changes
```

Reply:
can we change `base_trades` to `dex_base_trades` universally? while it only runs on dex CI, it still shows as a canceled step on other subprojects. this will help describe it better from an initial glance